src/config/ConfigManager.cpp

```diff
@@ -1691,13 +1691,16 @@ SConfigValue CConfigManager::getConfigValueSafeDevice(const std::string& dev, co
     const auto                  it = deviceConfigs.find(dev);
 
     if (it == deviceConfigs.end()) {
-        Debug::log(ERR, "getConfigValueSafeDevice: No device config for {} found???", dev.c_str());
-        return SConfigValue();
+        if (fallback.empty()) {
+            Debug::log(ERR, "getConfigValueSafeDevice: No device config for {} found???", dev.c_str());
```

Comment:
no need for .c_str() since format mr

Reply:
A lot of the formats in https://github.com/hyprwm/Hyprland/commit/61a71c65ace5124cb26a074715280537e6e34d00 use `.c_str()` so if that isn't the proper way to format strings it should be changed in a single commit.

Reply:
they do that cuz I was lazy

Reply:
I could write another pr for all of those, but I wouldn't put that in this pr. Just to keep it consistent.

Reply:
ye ok

Reply:
https://github.com/hyprwm/Hyprland/pull/3198#issue-1884627024